Fixtures/DependencyResolution/External/XCFramework/Package.swift

```diff
@@ -5,10 +5,14 @@ import PackageDescription
 let package = Package(
     name: "Foo",
     products: [
-        .library(name: "Foo", targets: ["Foo", "Bar"]),
+        .library(name: "Foo", targets: ["Foo", "Bar", "Baz"]),
```

Comment:
Mind setting up a separate test case for this? I'm worried that editing an existing fixture could have unintended effects on already existing test cases.

Reply:
No problem, I will add it to a new test case.

Reply:
I have setup a separate test case for this: `XCFrameworkExternal`, so we don't introduce unintended effects on existing test cases utilizing `XCFramework`.